src/ui/views/Dashboard/Header.tsx

```diff
@@ -346,6 +346,7 @@ const Header = ({ loading }) => {
 
     // Navigate if needed
     history.push('/dashboard');
+    window.location.reload();
```

Comment:
Need to add // eslint-disable-next-line no-restricted-globals above this. Not sure how it was checked in. We also shouldn't be accessing window directly but understand that it somehow works